src/idl_parser.cpp

```diff
@@ -278,6 +251,214 @@ static std::string TokenToString(int t) {
 }
 // clang-format on
 
+template<typename T>
```

Comment:
Is this large change because we moved everything lower in the file?

Reply:
Yes, the large changes are to group all the static/anonymous-namespace functions together.

I agree, it is harder to review, that's why I like just putting static at the front, but I agree the anonymous namespace is the way to go.

I suppose I could just add namespace { } around the functions where they stand...

Reply:
Lets just break it into two PRs. One adding static in place, then another followup PR moving to anonymous namespace. I think the diff would be clearer then.

Reply:
First PR done, static in place AND wrapped in little anonymous namespaces.
Grouping the little namespaces comes later.